File: desafiosIniciantes - Beecrowd/1020IdadeemDias.c
#include <stdio.h>
#include <math.h>
 
int main() {
 
    int old;
    
    scanf("%d", &old);
    printf("%.0f anos(s)\n", floor(old / 365));
    printf("%.0f mes(es)\n", floor(old % 365 / 30));
    printf("%.0f dia(s)\n", floor(old % 365 % 30));
 
    return 0;
}

File: desafiosIniciantes - Beecrowd/1020 - Idade em Dias.py
import math

old = int(input())

anos = math.floor(old / 365)
meses = math.floor((old % 365) / 30)
dias = math.floor(old % 365 % 30)

print(f"{anos:.0f} ano(s)")
print(f"{meses:.0f} mes(es)")
print(f"{dias:.0f} dia(s)")
